usabilla.py

```diff
@@ -300,20 +297,16 @@ def item_iterator(self, url):
 
         :type url: str
 
-        :returns: An `generator` that yeilds the requested data.
+        :returns: A `generator` that yeilds the requested data.
```

Comment:
Here, there's a typo in `yields` too.

Also this comment/docstring needs an addition, since the method now can also raise an `HTTPError`.